docs/superpowers/plans/2026-05-22-surface-parity-gap-closure.md

```
@@ -0,0 +1,1916 @@
+- Modify: `README.md`
+- Modify: `tests/test_live.sh`
+
+- [ ] **Step 1: Find REST API table in README**
+
+Run: `grep -n "^## REST API\|^### REST\|/api/" README.md | head -20`
+
+- [ ] **Step 2: Add 12 new REST routes to README table**
+
+Add entries for:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)